k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -982,7 +992,7 @@ public void refreshAttachmentThumbnail(AttachmentViewInfo attachment) {
         // mMessageView.refreshAttachmentThumbnail(attachment);
     }
 
-    private MessageCryptoMvpView messageCryptoMvpView = new MessageCryptoMvpView() {
+    private final MessageCryptoMvpView messageCryptoMvpView = new MessageCryptoMvpView() {
```

Comment:
I guess this is just optimization, right?

Reply:
yes.

Reply:
fixing some warnings.